conics/_conic.py

```diff
@@ -119,12 +130,12 @@ def adjugate(C):
     return np.linalg.inv(C).T * det
 
 
-def skew_symmetric(C):
+def skew_symmetric(C: np.ndarray) -> np.ndarray:
```

Comment:
`C` doesn't need to be a numpy array. It could be anything that can be unpacked to 3 scalars.

Reply:
I went with `Sequence[float]` which roughly captures this idea